cluster/manifests/roles/poweruser-binding.yaml

```diff
@@ -21,3 +21,11 @@ subjects:
 - kind: Group
   name: Emergency
   apiGroup: rbac.authorization.k8s.io
+{{- if eq .Cluster.ConfigItems.okta_auth_enabled "true" }}
+- kind: Group
+  name: "okta:common/engineer"
```

Comment:
Is this clear enough? Or should we rename it?

Reply:
I believe we already discussed it in the doc and we can live with that.

Reply:
I don't have any better proposals, so let's keep it like this.